e2e/screenshot.ts

```diff
@@ -0,0 +1,54 @@
+import * as fs from 'fs';
+import * as gulp from 'gulp';
+import * as path from 'path';
+import {browser} from 'protractor';
+
+
+function initializeEnvironment(jasmine: any) {
+  var reporter = new jasmine.JsApiReporter({});
+  reporter.specStarted = function(result: any) {
+    jasmine.getEnv().currentSpec = result.fullName;
```

Comment:
Changed to file-level variable. 

Constructor `JsApiReporter({specStarted})` is not working. 

Tested the test results are being reported normally.